src/ussd_higherhealth_healthcheck.js

```diff
@@ -78,18 +79,38 @@ go.app = (function () {
       return new MenuState(name, {
         question: $([
           "The HIGHER HEALTH HealthCheck is your risk assessment tool. Help us by answering a " +
-          "few questions about you and your health.",
+          "few questions.",
           "",
           "Reply"
         ].join("\n")),
-        error: $("This service works best when you select numbers from the list"),
+        error: $("Select a numbers from the list"),
         accept_labels: true,
         choices: [
-          new Choice("state_terms", $("START"))
+          new Choice("state_terms", $("START")),
+          new Choice("state_language", $("CHANGE LANGUAGE"))
         ]
       });
     });
 
+    self.add("state_language", function(name){
+        return new LanguageChoice(name, {
+        question: $([
+          "Select your language",
+          "",
+          "Reply:"
+        ].join("\n")),
+        accept_labels: true,
+        choices: [
+          new Choice("af", $("Afrikaans")),
+          new Choice("en", $("English")),
+          new Choice("st", $("Sotho")),
+          new Choice("xh", $("Xhosa")),
+          new Choice("zu", $("Zulu")),
+        ],
+        next: "state_terms"
+      });
+    });
```

Comment:
I think this should be the start screen, so the user selects their language first, and then the rest of the interaction is in that language.

Reply:
Then that means it would be mandatory for a user to always take that step, it would make sense if it is option, with English  being the default